packages/ListBox/src/components/Trigger/Trigger.styles.js

```diff
@@ -27,6 +27,12 @@ const triggerStyles = `
 `;
 
 const triggerStylesProps = () => props => {
+  if (props.isHidden) {
+    return css`
+      border: 1px solid #d7d7d7;
```

Comment:
Perhaps add a token for this hex value, seems to be currently be used in 4 paprika components

Reply:
this is part of the Listbox, right now this is not part of this PR. maybe it would be good to add an issue with this concern?